ui/app/components/app/modals/modal.js

```diff
@@ -464,61 +464,64 @@ function mapDispatchToProps (dispatch) {
   }
 }
 
-// Global Modal Component
-inherits(Modal, Component)
-function Modal () {
-  Component.call(this)
-}
+class Modal extends Component {
+  static propTypes = {
+    active: PropTypes.bool.isRequired,
+    onHideCallback: PropTypes.func,
+    hideModal: PropTypes.func.isRequired,
```

Comment:
This is missing the `hideWarning` prop. It's a function - massed through to the child `onHide` function and used on line 86 of this file.

Reply:
Well, that's an _unused_ prop since it's not unused in the component. I'm leaving that out so ESLint doesn't complain.

Reply:
Well it _is_ used because the props object is passed through. It's the same situation as when props are passed to a child using the spread operator. Just this ESLint rule doesn't seem to be catching this case. I'd consider it to be an ESLint bug.

You could change that signature to pass through `hideWarning` explicitly to make ESLint happy. That's the only prop ever used by that callback anyway.

Reply:
Updated